examples/finished/basiceven.py

```diff
@@ -0,0 +1,51 @@
+from pyscipopt import Model
+
+################################################################################
+# EVEN OR ODD?
+#
+# This example is made for newcomers and motivated by:
+# - modulus is unsupported for pyscipopt.scip.Variable and int
+# - negative integer numbers are not default
+# Based on this:
+# https://github.com/SCIP-Interfaces/PySCIPOpt/issues/172#issuecomment-394644046
+#
+################################################################################
+
+verbose = False
+sdic = {0:"even",1:"odd"}
+
+def parity(number):
+    assert number == int(round(number))
+    m = Model()
+    m.hideOutput()
+
+    ### "None" means -infinity as lower bound
+    ### N.B.: 0 is the default lower bound
+    ### thus negative numbers are not allowed
```

Comment:
what do you mean `thus negative numbers are not allowed` ? you can write `lb = -2`. Maybe is better to write
`m.addVar("x", vartype="I", lb = None, ub = None)` ?